theme/static/style/color-palette-light.css

```diff
@@ -57,3 +57,7 @@ nav.primary a.active, #pagination a.active {
     background-color: #6e827b;
     color: #e3edea;
 }
+
+.community h1 {
+    color: #9ab451;
+}
```

Comment:
We don't need new color for headings. Use the existing one and remove this style.